lib/hu_board/account_helpers.rb

```diff
@@ -25,6 +29,15 @@ def subscription_active?(customer)
       false
     end
 
+    def subscription_canceled?(customer)
+      cus = customer[:stripe][:customer]
+      if cus[:subscriptions][:total_count] > 0
+        sub = cus[:subscriptions][:data][0]
+        return sub[:status] == "canceled"
```

Comment:
Stripe uses "canceled" verses "cancelled", so I have been maintaining that convention in the app.